docker-database/hrmdb.sql

```diff
@@ -0,0 +1,924 @@
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."Contacts_id_seq" OWNER TO hrm;
+
+--
+-- Name: Contacts_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: hrm
+--
+
+ALTER SEQUENCE public."Contacts_id_seq" OWNED BY public."Contacts".id;
+
+
+--
+-- Name: SequelizeMeta; Type: TABLE; Schema: public; Owner: hrm
+--
+
+CREATE TABLE public."SequelizeMeta" (
+    name character varying(255) NOT NULL
+);
+
+
+ALTER TABLE public."SequelizeMeta" OWNER TO hrm;
+
+--
+-- Name: Subcategories; Type: TABLE; Schema: public; Owner: hrm
+--
+
+CREATE TABLE public."Subcategories" (
+    id integer NOT NULL,
+    subcategory character varying(255),
+    "createdAt" timestamp with time zone NOT NULL,
+    "updatedAt" timestamp with time zone NOT NULL
+);
+
+
+ALTER TABLE public."Subcategories" OWNER TO hrm;
+
+--
+-- Name: Subcategories_id_seq; Type: SEQUENCE; Schema: public; Owner: hrm
+--
+
+CREATE SEQUENCE public."Subcategories_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."Subcategories_id_seq" OWNER TO hrm;
+
+--
+-- Name: Subcategories_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: hrm
+--
+
+ALTER SEQUENCE public."Subcategories_id_seq" OWNED BY public."Subcategories".id;
+
+
+--
+-- Name: AgeBrackets id; Type: DEFAULT; Schema: public; Owner: hrm
+--
+
+ALTER TABLE ONLY public."AgeBrackets" ALTER COLUMN id SET DEFAULT nextval('public."AgeBrackets_id_seq"'::regclass);
+
+
+--
+-- Name: Contacts id; Type: DEFAULT; Schema: public; Owner: hrm
+--
+
+ALTER TABLE ONLY public."Contacts" ALTER COLUMN id SET DEFAULT nextval('public."Contacts_id_seq"'::regclass);
+
+
+--
+-- Name: Subcategories id; Type: DEFAULT; Schema: public; Owner: hrm
+--
+
+ALTER TABLE ONLY public."Subcategories" ALTER COLUMN id SET DEFAULT nextval('public."Subcategories_id_seq"'::regclass);
+
+
+--
+-- Data for Name: AgeBrackets; Type: TABLE DATA; Schema: public; Owner: hrm
+--
+
+COPY public."AgeBrackets" (id, bracket, min, max, "createdAt", "updatedAt") FROM stdin;
+1	0-3	0	3	2019-08-15 16:41:11.922+00	2019-08-15 16:41:11.922+00
+2	4-6	4	6	2019-08-15 16:41:11.922+00	2019-08-15 16:41:11.922+00
+3	7-9	7	9	2019-08-15 16:41:11.922+00	2019-08-15 16:41:11.922+00
+4	10-12	10	12	2019-08-15 16:41:11.922+00	2019-08-15 16:41:11.922+00
+5	13-15	13	15	2019-08-15 16:41:11.922+00	2019-08-15 16:41:11.922+00
+6	16-17	16	17	2019-08-15 16:41:11.922+00	2019-08-15 16:41:11.922+00
+7	18-25	18	25	2019-08-15 16:41:11.922+00	2019-08-15 16:41:11.922+00
+8	>25	26	120	2019-08-15 16:41:11.922+00	2019-08-15 16:41:11.922+00
+9	Unknown	\N	\N	2019-08-15 16:41:11.922+00	2019-08-15 16:41:11.922+00
+\.
+
+
+--
+-- Data for Name: Contacts; Type: TABLE DATA; Schema: public; Owner: hrm
+--
+
+COPY public."Contacts" (id, "timestamp", "taskId", "reservationId", "createdAt", "updatedAt", "AgeBracketId", "SubcategoryId", "rawJson", "queueName") FROM stdin;
```

Comment:
Do we want to include existing records? Can't that interfere with tests?

Reply:
The script is intended for general dev use, rather than automated tests - just to get something into the DB so REST calls to HRM start returning some data. Ideally you start with a fresh database for running tests, any data added during dev could affect them

Reply:
To clarify this script is never automatically run anywhere, developers have to choose to run it in pgAdmin or something - it's just to get them started playing around with the HRM service really